gradle/libs.versions.toml

```diff
@@ -0,0 +1,128 @@
+scripts-byteBuddy = { module = "net.bytebuddy:byte-buddy", version = "1.14.13" }
+selenium-apache-commons-lang3 = { module = "org.apache.commons:commons-lang3", version = "3.18.0" }
+soap-jakartaXmlSoapApi = { module = "jakarta.xml.soap:jakarta.xml.soap-api", version = "3.0.0" }
+soap-saajImpl = { module = "com.sun.xml.messaging.saaj:saaj-impl", version = "3.0.0" }
+soap-soaModelCore = { module = "com.predic8:soa-model-core", version = "2.0.1" }
+spider-kaitaiStructRuntime = { module = "io.kaitai:kaitai-struct-runtime", version = "0.10" }
+spiderAjax-apache-commons-math = { module = "commons-math:commons-math", version = "1.2" }
+spiderAjax-findBugsAnnotations = { module = "com.google.code.findbugs:jsr305", version = "3.0.2" }
+spiderAjax-guiceAssistedInject = { module = "com.google.inject.extensions:guice-assistedinject", version = "5.0.1" }
+spiderAjax-jcipAnnotations = { module = "net.jcip:jcip-annotations", version = "1.0" }
+spiderAjax-jclOverSlf4j = { module = "org.slf4j:jcl-over-slf4j", version.ref = "slf4j" }
+spiderAjax-julToSlf4j = { module = "org.slf4j:jul-to-slf4j", version.ref = "slf4j" }
+spiderAjax-metricsCore = { module = "com.codahale.metrics:metrics-core", version = "3.0.2" }
+spiderAjax-nekohtml = { module = "net.sourceforge.nekohtml:nekohtml", version = "1.9.22" }
+spiderAjax-slf4j-log4j12 = { module = "org.slf4j:slf4j-log4j12", version.ref = "slf4j" }
+spiderAjax-xmlunit = { module = "xmlunit:xmlunit", version = "1.6" }
+sqliplugin-jdom = { module = "org.jdom:jdom", version = "2.0.2" }
+test-hamcrest = { module = "org.hamcrest:hamcrest-library", version = "2.2" }
+test-junit-jupiter = { module = "org.junit.jupiter:junit-jupiter", version = "5.10.2" }
+test-junit-platformLauncher = { module = "org.junit.platform:junit-platform-launcher" }
+test-mockito-junit-jupiter = { module = "org.mockito:mockito-junit-jupiter", version = "5.10.0" }
+test-selenium-jupiter = { module = "io.github.bonigarcia:selenium-jupiter", version = "5.1.1" }
+testutils-httpclient5 = { module = "org.apache.httpcomponents.client5:httpclient5", version = "5.2.1" }
+testutils-nanohttpd-webserver = { module = "org.nanohttpd:nanohttpd-webserver", version.ref = "nanohttpd" }
+testutils-nanohttpd-websocket = { module = "org.nanohttpd:nanohttpd-websocket", version.ref = "nanohttpd" }
+testutils-zap = { module = "org.zaproxy:zap", version = "2.16.0" }
+wappalyzer-jsoup = { module = "org.jsoup:jsoup", version.ref = "jsoup" }
+wappalyzer-jsvg = { module = "com.github.weisj:jsvg", version = "1.4.0" }
+wappalyzer-re2j = { module = "com.google.re2j:re2j", version.ref = "re2j" }
+zest-jbrofuzzCore = { module = "org.owasp.jbrofuzz:jbrofuzz-core", version.ref = "jbrofuzz" }
+zest-zest = { module = "org.zaproxy:zest", version = "0.31.0" }
+
+[plugins]
+cyclonedx = { id = "org.cyclonedx.bom", version = "2.2.0" }
+datanucleus = { id = "org.rm3l.datanucleus-gradle-plugin", version = "2.0.0" }
+dependencyUpdates = { id = "com.github.ben-manes.versions", version = "0.52.0" }
```

Comment:
Do the tasks for this one still work with this setup?

Is its functionality superseded by these changes?

Reply:
Yes, that plugin not rely on where the dependencies are declared. No, upcoming dependabot changes might.

Reply:
Okay